_includes/manuals/1.20/1.2_release_notes.md

```diff
@@ -45,11 +45,12 @@ Foreman can now run properly in FIPS-enabled environments.
 #### API
 * API hosts/X/facts still inconsistent ([#25188](https://projects.theforeman.org/issues/25188))
 * Show hosts count in API ([#24782](https://projects.theforeman.org/issues/24782))
-* Add `hosts_count`, `can_edit/delete/create` in Models API response ([#24227](https://projects.theforeman.org/issues/24227))
+* Add hosts_count, can_edit/delete/create in Models API response ([#24227](https://projects.theforeman.org/issues/24227))
 * hammer fact list: 'organization' not recognized for searching! ([#23979](https://projects.theforeman.org/issues/23979))
 * API Support for easier use of Smart-Class Param Overrides ([#22090](https://projects.theforeman.org/issues/22090))
 
 #### Audit Log
+* for every fact import an audit record is written ([#25331](https://projects.theforeman.org/issues/25331))
```

Comment:
When I read this, it doesn't tell me whether this means that now it will always audit or ignore some. From the code I know it's the latter but you might want to clarify this

Reply:
I updated the issue title, next round will be clearer.